packages/frontend/src/components/accounts/batch_import_accounts/index.js

```diff
@@ -1,14 +1,15 @@
+import { decryptAccountData } from '@near-wallet-selector/account-export';
 import React, {useCallback, useEffect, useMemo, useState} from 'react';
 import { Translate } from 'react-localize-redux';
 import { useSelector } from 'react-redux';
 import styled from 'styled-components';
 import { useImmerReducer } from 'use-immer';
 
+
 import ShieldIcon from '../../../images/icon-shield.svg';
 import ImportArrow from '../../../images/import-arrow.svg';
 import { selectAccountUrlReferrer } from '../../../redux/slices/account';
 import { selectAvailableAccounts, selectAvailableAccountsIsLoading } from '../../../redux/slices/availableAccounts';
-import { decodeAccountsFrom } from '../../../utils/encoding';
```

Comment:
This function is now unreferenced, would you mind please deleting it and any other methods only referenced by it? This would be a small kindness to the new team who eventually takes this over 😄